debian/control

```diff
@@ -0,0 +1,63 @@
+Source: tsc
+Section: games
+Priority: optional
+Maintainer: Muammar El Khatib <muammar@debian.org>
+Build-Depends: debhelper (>= 9),
+               libcegui-mk2-dev (>= 0.6.2),
+               libboost-filesystem-dev,
+               libboost-thread-dev,
+               libboost1.61-all-dev,
+               libsdl1.2-dev,
+               libsdl-image1.2-dev,
+               libsdl-mixer1.2-dev,
+               libsdl-ttf2.0-dev,
+               pkg-config,
+               ttf-dejavu,
+               po4a,
+               gperf,
+               bison,
+               libxml++2.6-dev,
+               rake,
+               cmake,
+               libglew-dev,
+               ruby-full,
```

Comment:
`ruby-full` is not needed. The bare Ruby interpreter is enough, mruby even comes with its own Rake for building it (`minirake`). As long as you have the Ruby interpreter itself and its standard library available, that's enough. RubyGems, RDoc, Rake, etc. are not needed (unless you want to generate the documentation).